### PR TITLE
new stats badges

### PR DESCRIPTION
**Pull request by mpadge**
@joelnitta This is the announcment of your package and our new badges. ~~It includes an "author" file for you. I can't assign you as a reviewer here, but please add any additional information you like to your author file. Have a look at others for typical fields, which commonly include a one-line bio, and an ORCID id, and add any of those if you want. The author files link directly to internal pages, like [my one here](https://ropensci.org/author/mark-padgham/). Yours will be an initial placeholder, but with external links to the fields in your author file.~~

Update: Sorry Joel, author files only get started when you have your first internal page on the main website, such as a blog post. Link updated to your personal website instead.

### CODE REVIEW

File: content/author/joel-nitta/_index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: Joel Nitta
```
Comment (maelle) [inline content/author/joel-nitta/_index.md:2]: the author page will not appear when someone is not the author of any blog post / comm call so I'd suggest using the homepage link instead.

File: content/blog/2022-09-23-ropensci-news-digest-september-2022/index.Rmd [hunk 2]
```diff
@@ -58,6 +58,16 @@ You could volunteer on your own or as a team of maintainers (the more the merrie
 The rOpenSci team will be happy to help you with specifics, and will provide resources, tips and PR reviews as needed.
 Thank you!
 
+### New "Peer Reviewed" badges for Statistical Software Review
+
+We are thrilled to have the first package pass through our Statistical Software Review process.
```
Comment (maelle) [inline content/blog/2022-09-23-ropensci-news-digest-september-2022/index.Rmd:63]: we are already "thrilled" to launch the champions program in an earlier paragraph

```suggestion
We are overjoyed to have the first package pass through our Statistical Software Review process.
```

File: content/blog/2022-09-23-ropensci-news-digest-september-2022/index.Rmd [hunk 3]
```diff
@@ -58,6 +58,16 @@ You could volunteer on your own or as a team of maintainers (the more the merrie
...
+[The canaper package](https://github.com/ropensci/canaper) by [Joel NItta](/author/joel-nitta/) was [approved on the 14th of Sept. 2022](https://github.com/ropensci/software-review/issues/475#issuecomment-1247094903).
```
Comment (maelle) [inline content/blog/2022-09-23-ropensci-news-digest-september-2022/index.Rmd:64]:
```suggestion
[The canaper package](https://docs.ropensci.org/canaper/) by [Joel Nitta](https://www.joelnitta.com/) was [approved on the 14th of September 2022](https://github.com/ropensci/software-review/issues/475#issuecomment-1247094903).
```

File: content/blog/2022-09-23-ropensci-news-digest-september-2022/index.Rmd [hunk 4]
```diff
@@ -58,6 +58,16 @@ You could volunteer on your own or as a team of maintainers (the more the merrie
...
+Among many exciting aspects of the expansion of software peer review to statistical software is the introduction of new statistics-specific "Peer Reviewed" badges.
+Until now, every rOpenSci package has used the same single badge. Our new badges for statistical packages include a "grade" of [bronze, silver, or gold](https://stats-devguide.ropensci.org/pkgdev.html#pkgdev-badges), and the version number of the [*Statistical Software Standards*](https://stats-devguide.ropensci.org/standards.html) with which a package complies.
+You can see the first statistical software badge on the front page of [Joel's canaper repository](https://github.com/ropensci/canaper#canaper), with a "silver" grade for compliance with version 0.1 of our statistical standards.
```
Comment (maelle) [inline content/blog/2022-09-23-ropensci-news-digest-september-2022/index.Rmd:67]: I see a rotemplate bug, the badge should be on the pkgdown website too.

File: content/blog/2022-09-23-ropensci-news-digest-september-2022/index.Rmd [hunk 5]
```diff
@@ -58,6 +58,16 @@ You could volunteer on your own or as a team of maintainers (the more the merrie
...
+There are already quite a number of other statistical packages under review, and we'll be excited to see many more of these new badges appearing on rOpenSci packages soon.
```
Comment (maelle) [inline content/blog/2022-09-23-ropensci-news-digest-september-2022/index.Rmd:68]: link?

**Review by maelle — CHANGES_REQUESTED** [review]
Thank you!

File: content/blog/2022-09-23-ropensci-news-digest-september-2022/index.Rmd (hunk 5 above)
Comment (mpadge) [reply content/blog/2022-09-23-ropensci-news-digest-september-2022/index.Rmd]: done, thanks

File: content/author/joel-nitta/_index.md (hunk 1 above)
Comment (mpadge) [reply content/author/joel-nitta/_index.md]: author file removed

**Review by noamross — COMMENTED** [review]
LGTM!

**Comment by maelle** [comment]
oops, sorry for being too fast

**Pull request merged**